File: Zestawy Zadań/C++/4 - Tablice/Zestaw 1 zadania/4.1.3 - Tablica dwuwymiarowa liczby 1 po przekątnej.cpp
#include <iostream>

using namespace std;

int main()
{
    int dane[10][10], sum = 0;
    for (int i = 0; i < 10; i++)
    {
        for (int j = 0; j < 10; j++)
        {
            if (i == j)
            {
                dane[i][j] = 1;
                sum += dane[i][j];
            }
            else
            {
                dane[i][j] = 0;
            }
            cout << dane[i][j] << " ";
        }
        cout << endl;
    }
    cout << endl;
    cout << "Suma: " << sum << endl;
    return 0;
}

File: Zestawy Zadań/Python/4 - Tablice/Zestaw 1 zadania/4.1.3 - Tablica dwuwymiarowa liczby 1 po przekątnej.py
dane = [[0] * 10 for _ in range(11)]
sum = 0

for i in range(10):
    for j in range(10):
        if i == j:
            dane[i][j] = 1
            sum += dane[i][j]
        else:
            dane[i][j] = 0
        print(dane[i][j], end=" ")
    print()

print("Suma:",sum)